emperor/_pandas.py

```diff
@@ -0,0 +1,103 @@
+# ----------------------------------------------------------------------------
+# Copyright (c) 2013--, emperor development team.
+#
+# Distributed under the terms of the Modified BSD License.
+#
+# The full license is in the file LICENSE.md, distributed with this software.
+# ----------------------------------------------------------------------------
+from __future__ import division
+
+import numpy as np
+import pandas as pd
+
+from emperor.core import Emperor
+from skbio import OrdinationResults
+
+
+def scatterplot(df, x=None, y=None, z=None, remote=True):
+    """Create an Emperor scatter plot from a Pandas DataFrame
+
+    Parameters
+    ----------
+    df : pd.DataFrame
+        Pandas DataFrame with the data to display, this includes both
+        *metadata* and coordinates to position the samples in a 3D space.
+    x, y, z : str, optional
+        Column names in `df`, to use as first (``x``), second (``y``) and third
+        (``z``) axes in the visualization. If these are not specified, axes
+        are chosen according to the variance (in decremental order).
+    remote : bool, optional
+        Whether the JavaScript resources should be loaded locally or from
+        GitHub. Defaults to ``True``.
+
+    Returns
+    -------
+    emperor.core.Emperor
+        Emperor object with the numerical data as the `ordination` attribute
+        and the entire DataFrame as the `mf` attribute.
+
+    Raises
+    ------
+    ValueError
+        If `df` is not a PandasDataFrame
+        If `x`, `y` or `z` are missing from `df` or if they are not numeric
+        columns.
+        If after removing rows with missing data there are fewer than 3
+        samples.
+
+    Notes
+    -----
+    If a row has missing data, that data pont will be removed from the
+    visualization.
+
+    See Also
+    --------
+    emperor.core.Emperor
+    """
+
+    if not isinstance(df, pd.DataFrame):
+        raise ValueError("The argument is not a Pandas DataFrame")
+
+    for col in [z, y, x]:
+        if col is None:
```

Comment:
These can all be `None`, if they are not explicitly specified, then the columns with the highest variance will take precedence and take the place of `x`, `y` or `z` correspondingly. That's why I chose to make them optional arguments.